src/styles/theme.js

```diff
@@ -77,9 +77,10 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
     alignItems: 'stretch',
   },
   link: {
-    marginTop: 10,
+    paddingTop: 10,
     fontSize: 15,
     color: BRAND_COLOR,
+    textAlign: 'right',
```

Comment:
Actually It was clickable on the master too. If you can check, on iOS and Android both were clickable.